docs-ref-conceptual/azure-cli-vm-tutorial-2.md

```diff
@@ -12,11 +12,11 @@ ms.custom: devx-track-azurecli, seo-azure-cli
 keywords: azure cli create vnet, virtual network in azure cli, subnet in virtual network
 ---
 
-# 2 - Create a virtual network
```

Comment:
@daphnemamsft , Tutorial H1s should use the numer `2` not the word `two`.  See this Python tutorial for an example: https://learn.microsoft.com/en-us/azure/developer/python/tutorial-containerize-deploy-python-web-app-azure-02?tabs=sample-app-git-clone%2Cvscode-docker%2Cterminal-bash%2Cmongodb-local

Reply:
Sounds good, I will go ahead and revise this and take this into account for the other docs